docs/multiarch.md

```diff
@@ -0,0 +1,58 @@
+# How do I ship an operator that supports multiple node architectures?
+
+It is assumed that all operators run on linux hosts, but can be configured to manage workloads on other architectures if worker nodes are available.
+
+Which os/arches are supported by the operator can be indicated by adding labels to the ClusterServiceVersion providing the operator.
+
+## Supported Labels
+
+Labels indicating supported os and arch are defined by:
+
+```yaml
+labels:
+    operatorframework.io/arch.<GOARCH>: supported
+    operatorframework.io/os.<GOOS>: supported
+```
+
+Where `<GOARCH>` and `<GOOS>` are one of the values [listed here](https://github.com/golang/go/blob/master/src/go/build/syslist.go).
+
+## Multiple Architectures
+
+Some operators may support multiple node architectures or oses. In this case, multiple labels can be added. For example, an operator that support both windows and linux workloads:
+
+```yaml
+labels:
+    operatorframework.io/os.windows: supported
```

Comment:
Can the label value be something other than `supported`?

Reply:
See also: https://github.com/operator-framework/olm-book/pull/29#discussion_r365452635